test/test_log_prob.jl

```diff
@@ -110,5 +116,5 @@ end
 test_sigmoid_logit()
 test_gal_shape_constrain()
 test_color_flux_transform()
-#test_that_star_truth_is_most_likely_log_prob()
+test_that_star_truth_is_most_likely_log_prob()
 #test_that_gal_truth_is_most_likely()
```

Comment:
or `test_that_gal_truth_maximizes_log_prob`, that name should be unique. Ryan added `_log_prob` just to avoid naming conflicts with the elbo tests.